curation_concerns-models/app/jobs/upload_set_update_job.rb

```diff
@@ -26,20 +26,8 @@ def perform(login, upload_set_id, title, file_attributes, visibility)
     upload_set.update(status: ["Complete"])
 
     if denied.empty?
-      unless saved.empty?
```

Comment:
I don't think we should wipe out the blocks of code in this method. Or, at very least, if we do, we should ticket adding them back. We use these callbacks in Sufia to send notifications.